README.md

```diff
@@ -10,7 +10,7 @@ support new services is also a fairly simple process.
 
 ## Table of contents
 1. [Installation](#installation)
-2. [Quick setup from a template](#Setup from a template)
+2. [Quick setup from a template](#Setup_from_a_template)
```

Comment:
Links have `-` in b/w words, not `_`. Fixed it via https://github.com/AMP-SCZ/lochness/commit/ceca5eeb3db3d47196fa10d5d095b970dd9862ca